server.js

```diff
@@ -90,7 +90,7 @@ app.set('view engine', 'html')
 
 // Middleware to serve static assets
 app.use('/public', express.static(path.join(__dirname, '/public')))
-app.use('/assets', express.static(path.join(__dirname, 'node_modules', 'govuk-frontend', 'assets')))
+app.use('/node_modules/govuk-frontend/assets', express.static(path.join(__dirname, 'node_modules', 'govuk-frontend', 'assets')))
```

Comment:
I think we should probably change this in the guidance before changing it in the prototype kit?

Reply:
yeah it terms of https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#recommended-solution
if they mount just /assets, nothing else needs to change

if you choose a different path, you need to redefine variable value, like in this PR. 

Not sure what is easier to comprehend for someone trying to do this

Reply:
Think it might be better to do this kinda thing based on research, since we're doing things differently here than what we recommend at install.

Reply:
couldn't our current advice lead to a clash? If people are already using `/assets`? Something with a namespace might be safer like `/govuk-frontend/assets`

Reply:
it can yes

Reply:
see linked PR in Frontend to make the change